ytmusicapi/parsers/podcasts.py

```diff
@@ -103,6 +103,7 @@ def parse_episodes(results) -> List[Dict]:
     episodes = []
     for result in results:
         data = nav(result, ["musicMultiRowListItemRenderer"])
+        thumbnails = nav(data, THUMBNAILS)[:2]
```

Comment:
why are you only taking the first two items? just leave out the `[:2]`

Reply:
Some podcasts episodes appear to contain technical images in addition to the episode related images, e.g.
https://i.ytimg.com/vi/dPw_SXs1QOA/hq720.jpg:
Podcast 'Smosh Dungeons and Dragons', beginning of first episode:
`podcast = [{
        'musicMultiRowListItemRenderer': {
            'trackingParams': 'CHEQ3-AIGAAiEwjssYvPvceEAxWU0UIFHRKZBZk=',
            'thumbnail': {
                'musicThumbnailRenderer': {
                    'thumbnail': {
                        'thumbnails': [{
                                'url': 'https://i.ytimg.com/vi/f-vhvYvI0ok/hqdefault.jpg?sqp=-oaymwEWCOADEI4CIAQqCggAEOADGC0guwJIWg&rs=AMzJL3lYQ_uibdN_dDCw6PWruHiO_dolmQ',
                                'width': 480,
                                'height': 270
                            }, {
                                'url': 'https://i.ytimg.com/vi/f-vhvYvI0ok/hq720.jpg?sqp=-oaymwEKCKAGEMIDIABIWg&rs=AMzJL3k6ozV_-vU-izDtPVWu-lbKy5wgAw',
                                'width': 800,
                                'height': 450
                            }, {
                                'url': 'https://i.ytimg.com/vi/f-vhvYvI0ok/hq720.jpg',
                                'width': 1280,
                                'height': 720
                            }
                        ]
                    },
                    'thumbnailCrop': 'MUSIC_THUMBNAIL_CROP_UNSPECIFIED',
                    'thumbnailScale': 'MUSIC_THUMBNAIL_SCALE_ASPECT_FIT',
                    'trackingParams': 'CHsQhL8CIhMI7LGLz73HhAMVlNFCBR0SmQWZ'
                }
            },
`
Usually I display the last image in the list as it normally has the best resolution. But here that would result in the wrong image being displayed. So far, I haven't found episodes where such a technical image appeared before third position. Not having come up with a better idea so far, I therefore had decided to drop all thumbnails after position two.
Just now I realize that the third image's URL doesn't contain a question mark, so an alternative approach would be to filter for thumbnails with URLs containing one:
thumbnails = list(filter(lambda tn: '?' in tn['url'], nav(data, THUMBNAILS)))

Reply:
I realize that the same image is present in multiple resolutions. This is the case for other thumbnails as well.

`ytmusicapi` always returns all thumbnails - it's up to the user to choose the appropriate size. You never know the precise use case in advance, so can't be too judgy on the input data filtering.